### DISCUSSION TITLE
Demo code for the physics test problems in the SCBO paper?

### DISCUSSION THREAD

**Discussion by Ted12345678** [meta-pytorch/botorch#1787] [Q&A]
I am studying the paper in the SCBO tutorial reference, [1]: David Eriksson and Matthias Poloczek. Scalable constrained Bayesian optimization. In International Conference on Artificial Intelligence and Statistics, pages 730–738. PMLR, 2021. (https://doi.org/10.48550/arxiv.2002.08526).

Anybody tried to repeat the physics test problems, for example, 3D tension-compression string problem? Is there a demo code available? Any suggestions if try to plug the test problem to the SCBO tutorial code?

Thanks.

**Comment by Balandat** [comment]
cc @dme65

**Comment by Ted12345678** [comment]
If anybody could explain how to do the following part from the paper, it would be very helpful:

To the objective function, we apply
a Gaussian copula (e.g., see Wilson & Ghahramani, 2010).
The Gaussian copula first maps all observations under the
objective to quantiles using the empirical CDF. Then it maps
the quantiles through an inverse Gaussian CDF. 

Thank you very much!
